test/unit/src/common/actions/t_create-snap.js

```diff
@@ -73,46 +73,43 @@ describe('create snap actions', () => {
     });
 
     afterEach(() => {
+      scope.done();
       nock.cleanAll();
     });
 
-    it('throws an error on failure to create webhook', () => {
+    it('throws an error on failure to create webhook', async () => {
       scope
         .post('/api/github/webhook', { owner: 'foo', name: 'bar' })
         .reply(404, {
           status: 'error',
           payload: { code: 'github-repository-not-found' }
         });
-      return createWebhook(repository)
-        .then(() => { throw new Error('unexpected success'); })
-        .catch((error) => {
-          scope.done();
-          expect(error.json.payload.code).toEqual(
-            'github-repository-not-found'
-          );
-        });
+      try {
+        await createWebhook(repository);
+        throw new Error('unexpected success');
+      } catch (error) {
+        expect(error.json.payload.code).toEqual('github-repository-not-found');
+      }
     });
 
-    it('succeeds if creating webhook says already created', () => {
+    it('succeeds if creating webhook says already created', async () => {
       scope
         .post('/api/github/webhook', { owner: 'foo', name: 'bar' })
         .reply(422, {
           status: 'error',
           payload: { code: 'github-already-created' }
         });
-      return createWebhook(repository)
-        .then(() => scope.done());
+      await createWebhook(repository);
```

Comment:
It's a bit confusing that there is no `expect` in this test at all.

But I guess it expects not to throw any error...